pyo3-ffi/src/weakrefobject.rs

```diff
@@ -59,4 +59,7 @@ extern "C" {
     pub fn PyWeakref_NewProxy(ob: *mut PyObject, callback: *mut PyObject) -> *mut PyObject;
     #[cfg_attr(PyPy, link_name = "PyPyWeakref_GetObject")]
     pub fn PyWeakref_GetObject(_ref: *mut PyObject) -> *mut PyObject;
```

Comment:
```suggestion
    #[cfg_attr(Py_3_13, deprecated(note = "deprecated since Python 3.13. Use `PyWeakref_GetRef` instead."))]
    pub fn PyWeakref_GetObject(ref: *mut PyObject) -> *mut PyObject;
```